### PR TITLE
zh-CN: create `management.uninstall()`

### PR DESCRIPTION
**Pull request by PassionPenguin**

### CODE REVIEW

**Review by PassionPenguin — COMMENTED** [review]
awaiting mdn/content#37091

**Comment by jasonren0403** [comment]
Let's convert to draft first

**Comment by yin1999** [comment]
> awaiting [mdn/content#37091](https://github.com/mdn/content/pull/37091)

The upstream PR has been merged. Please check the translation again and Mark it as ready for review.

File: files/zh-cn/mozilla/add-ons/webextensions/api/management/uninstall/index.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+title: management.uninstall()
+slug: Mozilla/Add-ons/WebExtensions/API/management/uninstall
+l10n:
+  sourceCommit: b8a0743ca8b1e1b1b1a95cc93a4413c020f11262
```
Comment (jasonren0403) [inline files/zh-cn/mozilla/add-ons/webextensions/api/management/uninstall/index.md:5]:
```suggestion
  sourceCommit: c9bad2ef8b5655146481292f3bce7de0dd86ba68
```

**Review by jasonren0403 — APPROVED** [review]
👍

**Pull request merged**